### PR TITLE
Mailmotor SubscribeType/UnsubscribeType: ucfirst was missing in label

### PR DESCRIPTION
**Pull request by jeroendesloovere**
## Type

- Enhancement

## Pull request description

In the Frontend Mailmotor templates, the button label did not had a capital letter.
This PR fixes that.

### CODE REVIEW

File: src/Frontend/Modules/Mailmotor/Domain/Subscription/SubscribeType.php [hunk 1]
```diff
@@ -76,7 +76,7 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             'subscribe',
             SubmitType::class,
             [
-                'label' => 'lbl.Subscribe',
+                'label' => ucfirst(Language::lbl('Subscribe')),
```
Comment (carakas) [inline src/Frontend/Modules/Mailmotor/Domain/Subscription/SubscribeType.php]: wrong place to fix this, this should be, if needed fixed in the form templates not here

**Comment by carakas** [comment]
Fixed by https://github.com/forkcms/forkcms/pull/2323

**Comment by jeroendesloovere** [comment]
Did you refer to a wrong PR?
I just wanted to have "Send" instead of "send" in the button.
The other text labels do have "ucfirst", so why can't is use ucfirst on that button label?

**Comment by carakas** [comment]
Most things use SpoonFilter:ucfirst because that is multi byte safe

**Comment by carakas** [comment]
but apart from that, this is symfony form, so shouldn't this be fixed in the form template so it is fixed for all buttons

**Pull request merged**